docs/source/documentation.html.md.erb

```diff
@@ -655,12 +655,27 @@ Latitude and longitude of the place found. Shape: `{lat: 48.797885, lng: 2.33703
 Type: **string**
 </td>
 <td markdown="1">
-Postcode (or ZIP Code) of the place found.
+The best matching postcode (or ZIP Code) of the place found.
 
 Examples:
 
- - 94230
- - 1001
+ - 94104
+ - 75009
+</td>
+    </tr>
+    <tr>
+<td markdown="1">
+<div class="api-entry" id="api-suggestion-postcodes"><code>postcodes</code></div>
+
+Type: **array**
+</td>
+<td markdown="1">
+The list of all postcode (or ZIP Code) of the place found.
+
+Examples:
+
+ - `["94102","94103","94104",...]`
```

Comment:
Format with spaces

Reply:
Now is:

![image](https://user-images.githubusercontent.com/5136989/47726358-9fbe0d00-dc5a-11e8-9ea0-76e3dd18bd76.png)

which was the way I felt it rendered the best